src/network/networkd-setlink.c

```diff
@@ -16,6 +16,7 @@ static const char *const set_link_mode_table[_SET_LINK_MODE_MAX] = {
         [SET_LINK_FLAGS] = "link flags",
         [SET_LINK_GROUP] = "interface group",
         [SET_LINK_MAC] = "MAC address",
+        [SET_LINK_MASTER] = "master interface",
         [SET_LINK_MTU] = "MTU",
```

Comment:
i think @keszybz prefers if we align the `=` in the same columns for enum tables like this ;-)

doesn't matter though